core/services/AuthService.js

```diff
@@ -0,0 +1,12 @@
+const bcrypt = require("bcrypt");
+const saltRounds = 10;
+
+module.exports = {
+  encryptPassword (password) {
+    return bcrypt.hashSync(password, saltRounds);
+  },
+
+  checkPassword (password, passwordHash) {
+    return bcrypt.compareSync(password, passwordHash);
+  }
+}
```

Comment:
There is a standard requirement to have the empty line at the end of each file. You can set this option in your Webstorm config, so it will do it automatically.